bin/PerlDDS/Run_Test.pm

```diff
@@ -420,6 +422,12 @@ sub new {
     my $flag_value_str = defined($flag_value) ? "\"$flag_value\"" : 'undef';
     $self->_info("TestFramework storing \"$flag_name\"=$flag_value_str\n");
     $self->{_flags}->{$flag_name} = $flag_value;
+
+    if (exists $self->{configs}->{$flag_name}) {
+        $self->_info("TestFramework switching to config $flag_name\n");
+        $self->{config} = $flag_name;
```

Comment:
I know this and many Perl files aren't consistent, but [we should prefer 2 spaces to 4 spaces](https://opendds.readthedocs.io/en/master/internal/dev_guidelines.html#perl-coding-style).